src/components/Comments/WriteComment.js

```diff
@@ -17,6 +18,7 @@ import StringConstant from '../../utils/string/StringConstant';
 import {Context} from '../../context';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good addition of `DEFAULT_PROFILE_PIC_PATH` import. This will be useful for providing a default profile picture when none is available.



<!-- This is an auto-generated comment by OSS CodeRabbit -->